formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/aws/DynamoClient.scala

```diff
@@ -22,7 +22,12 @@ trait DynamoClient {
   def deleteUserSubmissions(submissionIdsAndEmails: List[SubmissionIdEmail], salt: String, submissionsTableName: String): Either[Throwable, List[DeleteItemResult]]
 }
 
-case class SubmissionTableUpdateDate(formstackSubmissionTableMetadata: String, date: String)
+case class SubmissionTableUpdateDate(formstackSubmissionTableMetadata: String, date: String){
+  def toEasternTime:String = {
+    val utcDate = LocalDateTime.parse(date, formatter).atZone(ZoneOffset.UTC)
+    utcDate.withZoneSameInstant(ZoneId.of("America/New_York")).format(formatter)
+  }
```

Comment:
I think this date is just a string because it's read from dynamodb and then directly used to pass in to formstack but now we have to parse it into a date to change the timezone and then parse change it back into a string.

Another alternative would be to always use eastern time everywhere but I thought it is expected that dates are in UTC so maybe it's less confusing to translate it into eastern just when we pass it to formstack but leave it as UTC everywhere else